cluster/gce/configure-vm.sh

```diff
@@ -762,7 +762,7 @@ users:
 clusters:
 - name: local
   cluster:
-    server: https://kubernetes-master
+    server: https://${1}:443
```

Comment:
git grep create-salt-kubelet-auth

```
cluster/gce/configure-vm.sh:function create-salt-kubelet-auth() {
cluster/gce/configure-vm.sh:  create-salt-kubelet-auth "${KUBERNETES_MASTER_NAME}"
cluster/vagrant/provision-master.sh:  create-salt-kubelet-auth
cluster/vagrant/provision-node.sh:  create-salt-kubelet-auth
cluster/vagrant/provision-utils.sh:function create-salt-kubelet-auth() {
```

And since Vagrant's function is independent, the only call site for this function in the core repo is the one in this file.